fixtures/higherhealth_university_province.csv

```diff
@@ -88,3 +88,4 @@ ZA-WC,False Bay,86
 ZA-WC,Northlink,87
 ZA-WC,South Cape,88
 ZA-WC,West Coast,89
+,Other,90

```

Comment:
I see we're still adding things to the database here. I think this should rather be added in `higher_health/templates/healthcheck_questionnaire.html`, on line 166 and 172, where we've got the `let campuses = ...` and `let universities = ...`, we have the list of items from the database, and then we add the `Other` item ourselves.

Reply:
@rudigiesler it makes more sense to have it defined in the backend/db, if we let the FE handle it, it adds more code to be written which will then introduce more bugs

Reply:
The issue with having it in the database, is that now we have this special hard-coded database ID number. That makes it very easy for something to break, much easier than something breaking in the front-end.

We can always add unit and integration tests for the front-end.

Reply:
@rudigiesler this I can assure you will cause a lot of headaches this then means adding more code, while we already have predefined values of the data. Ideally the form instance should be a django model form, with some some mixin to order and manipulate the display logic